spec.emu

```diff
@@ -144,6 +144,14 @@ location: https://github.com/tc39/proposal-decimal/
 	<p>If _options_ is specified and _roundingMode_ is a property of _options_, then Get(_options_, *roundingMode*) must be a rounding mode. If Get(_options_, *roundingMode*) is not a rounding mode, a *RangeError* will be thrown. The multiplication will be carried out according to the specified rounding mode.</p>
       </emu-note>
     </emu-clause>
+    <emu-clause id="sec-decimal-method-fused-multiply-and-add">
```

Comment:
This is just a rough spec text for stage 2 so these IDs probably don't matter, but for future reference in stage 3 it will need to be in the format `sec-decimal.prototype.multiplyandadd`

Reply:
Ah, thanks -- I was aware of that but didn't know what the convention was.